java/runtime/src/main/java/io/ray/runtime/RayNativeRuntime.java

```diff
@@ -102,12 +102,14 @@ public void start() {
 
   @Override
   public void shutdown() {
-    nativeShutdown();
-    if (null != manager) {
-      manager.cleanup();
-      manager = null;
+    if (rayConfig.workerMode == WorkerType.DRIVER) {
```

Comment:
could you add a comment here on that only driver is allowed to call native shutdown?